ledfx/effects/audio.py

```diff
@@ -307,7 +331,6 @@ def open_audio_stream(device_idx):
                 self._stream = self._audio.InputStream(
                     samplerate=int(device["default_samplerate"]),
                     device=device_idx,
-                    channels=ch,
```

Comment:
I am worried about removing this param, on my machine it hard crashes my audio and the only recovery is a full system reboot, moving back to the code with channels param does not fix without the full system reboot.

I can try this PR specifically in case I have other side effects.

Reply:
ignore this, I cannot reproduce that specific failure when running this branch cleanly